apps/docs/pages/guides/auth/sessions.mdx

```diff
@@ -26,6 +26,7 @@ A session terminates, depending on configuration, when:
 - The user changes their password or performs a security sensitive action.
 - It times out due to inactivity.
 - It reaches its maximum lifetime.
+- A user signed in on another device.
```

Comment:
```suggestion
- A user signs in on another device.
```

I'd suggest putting the condition here (`when "single user per session" is configured` or something like that), as it's pretty easy to misread this as the default behavior, which might not be what many people want. (Might apply to the other items as well)

Reply:
It has a sentence above it that says "depending on configuration"